sphinxcontrib/inmanta/api.py

```diff
@@ -54,23 +61,23 @@ def format_multiplicity(rel):
     return str(low) + ":" + str(high)
 
 
-def parse_docstring(docstring):
+def parse_docstring(docstring: str) -> dict[str : Union[dict[str, str], list[str]]]:
```

Comment:
```suggestion
def parse_docstring(docstring: str) -> dict[str, Union[dict[str, str], list[str]]]:
```
Is this what you meant here?